x/margin/keeper/close.go

```diff
@@ -31,20 +31,22 @@ func (k Keeper) Close(ctx sdk.Context, msg *types.MsgClose) (*types.MsgCloseResp
 		return nil, sdkerrors.Wrap(types.ErrInvalidPosition, mtp.Position.String())
 	}
 
+	collateralIndex := len(mtp.Collaterals) - 1
```

Comment:
Is there any reason using last one for events?

Reply:
@kenta-elys can you comment on that, I reproduced what you did for open events

Reply:
@jelysn @kenta-elys I will add a change in my next PR to include all the denoms in the event attributes